### PR TITLE
Ft login express

### PR DESCRIPTION
**Pull request by muhireheir**
> #### What does this PR do?

> * validate login  form
> * log user in
> #### Description of Task to be completed
> * checks if user entered correct data format and if the form is not empty
> #### How should this be manually tested?
>  * clone the repository
> * change to ft-login-express
> * go into server folder
> * open  terminal  in server folder
> * run  the  following   codes `npm start`
> * open postman
> * i  url enter the following adress `localhost:300/signin`
> * send request with post method
> * see  the image below  to have an idea of params must be in the body of the [request](url)
> * Done.
> #### Any background context you want to provide?
> * allowed user with account to lig in
> #### What are the relevant Trello story?
>  * https://trello.com/c/8NhIhNzK
> #### Screenshots (if appropriate)
logi

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let http = require('http')
+let port=process.env.PORT || 3000
+let app = require('./app')
+let server=http.createServer(app)
+server.listen(3000)
```
Comment (habinezadalvan) [inline server/server.js:5]: Could you please add a second parameter of a call-back function that console logs the port your server is running on. Currently, someone can not know whether the server is on or not.

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "atlp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js"
```
Comment (habinezadalvan) [inline server/package.json:8]: You're using Nodemon yet it's not in your dev-dependencies or dependencies. Someone without this package installed globally on the local machine can not run your project locally. Could you please add it to your dev-dependencies.

File: server/api/contollers/signInController.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            })
+        }
+        
+    }else{
+        res.status(400).json({
+            error:{message:'Missing Required Params'}
+        })
+    }
+    
+}
```
Comment (habinezadalvan) [inline server/api/contollers/signInController.js:22]: Great work, I like the way you're using controllers and routes and validating the user. Anyway, is it possible for you to use dummy data or mongoose and login with stored data?

**Review by habinezadalvan — CHANGES_REQUESTED** [review]
Nice work so far. I just dropped a few comments on this PR. Please work on them before you merge the work.

File: server/package.json (hunk 2 above)
Comment (mukunzidd) [reply server/package.json:8]: @muhireheir 
Can you resolve the conflicts, add `node_modules` to `.gitignore`  add `nodemon` to the devDependencies?

**Pull request closed without merging**